microsoft-365/admin/manage/set-password-expiration-policy.md

```diff
@@ -69,9 +69,12 @@ Here are some things to know about how this feature currently works as of Januar
     
 - Users do not get an email notification that their password is going to expire in X number of days. Do you want this feature? **[Vote here!](https://office365.uservoice.com/forums/273493-office-365-admin/suggestions/15028344-office-365-password-email-notification)**
     
-## Prevent last password from being used again
+## Prevent last password from being used again.
 
-If you want to prevent your users from recycling old passwords, you can do so by enforcing password history in Azure AD. See [Create a custom password policy](https://docs.microsoft.com/azure/active-directory-domain-services/password-policy#create-a-custom-password-policy).
+If you want to prevent your users from recycling old passwords, you can do so by enforcing password history in On Premise AD. See [Create a custom password policy](https://docs.microsoft.com/azure/active-directory-domain-services/password-policy#create-a-custom-password-policy).
```

Comment:
```suggestion
If you want to prevent your users from recycling old passwords, you can do so by enforcing password history in on-premises Active Directory (AD). See [Create a custom password policy](https://docs.microsoft.com/azure/active-directory-domain-services/password-policy#create-a-custom-password-policy).
```